stages/org.osbuild.sudoers

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/python3
+"""
+Create a sudoers file
+
+This stage accepts a list of user specifications. Each user specification
+consists of a `users` array, a `hosts` array, and a `command definition` array.
+"""
+
+import os
+import sys
+
+import osbuild.api
+
+
+SCHEMA = r"""
+  "definitions": {
+    "user": {
+      "type": "string",
+      "pattern": "^[\\w]+$"
```

Comment:
@ondrejbudai recently fixed the stages to allow dots in user-names. I think we should make sure not to break that again.

```
commit 092f88690451cc83fe22bf6f1d6c9b5b88000c61
Author: Ondřej Budai <ondrej@budai.cz>
Date:   Wed Jul 20 10:20:39 2022 +0200

    stages/users: support a dot inside a username
```

Reply:
Nice find, I switched the pattern to be the same as the one in the `org.osbuild.users` stage so a user added by that stage can always be added to sudoers.